src/renderer/components/+catalog/catalog.tsx

```diff
@@ -252,10 +252,10 @@ export class Catalog extends React.Component<Props> {
           { title: "Labels", className: css.labelsCell, id: "labels" },
           { title: "Status", className: css.statusCell, sortBy: sortBy.status, id: "status" },
         ]}
-        customizeTableRowProps={(item: CatalogEntityItem<CatalogEntity>) => ({
+        customizeTableRowProps={item => ({
           disabled: !item.enabled,
         })}
-        renderTableContents={(item: CatalogEntityItem<CatalogEntity>) => [
+        renderTableContents={item => [
```

Comment:
nit: change `item` to `entity`, to be consistent?

Reply:
Yes